cli/src/pcluster/config/cluster_config.py

```diff
@@ -622,6 +622,11 @@ def __init__(self, subnet_ids: List[str], assign_public_ip: str = None, **kwargs
         self.assign_public_ip = Resource.init_param(assign_public_ip)
         self.subnet_ids = Resource.init_param(subnet_ids)
 
+    @property
+    def queue_subnet_id_az_mapping(self):
```

Comment:
NIT: Following the naming convention we use, this should be `subnet_id_az_mapping` since it's already in the context of "Queue" Networking.

Reply:
Done.